data/competencies.yml

```diff
@@ -510,6 +510,8 @@
   summary: Improves delivery process and encourages others to do the same
   examples:
     - Updates the scrum process to fit the changing needs of the team
+    - Champions technical issues that effect delivery such as release cycles, dealing with tech debt and bug fixes
```

Comment:
This should be "affect" not "effect"

Reply:
... Yes... I totally set you up for this because it was me that wrote the copy originally 🙈

Reply:
👋 🤜

Reply:
(waving fist, not threatening violent retribution... although arguably waving a fist is that. Where's that retreat emoji when I need it)